package.json

```diff
@@ -15,7 +15,8 @@
     "format": "prettier --write '**/*.{js,jsx,ts,tsx}'"
   },
   "devDependencies": {
-    "prettier": "^2.3.1"
+    "prettier": "^2.3.1",
+    "standard": "^16.0.3"
```

Comment:
These two are not compatible. We should choose one. My preference is standard, because it lints as well as formats and there's plugins that can show warnings in your editor as they occur.

Reply:
i'm gonna revert this change for this PR, so we can land the linting change as a seperate thing.

Reply:
Done